### PR TITLE
[FEAT] Pineapple Shirt

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

This PR adds the Pineapple shirt NPC for the smoothie shack

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/types/bumpkin.ts [hunk 1]
```diff
@@ -187,6 +189,8 @@ export const ITEM_IDS: Record<BumpkinItem, number> = {
   "Mountain View Background": 78,
   "Skull Hat": 79,
   "Reindeer Antlers": 80,
+  "Santa Hat": 81,
+  "Pineapple Shirt": 82,
```
Comment (mGranTY) [inline src/features/game/types/bumpkin.ts:193]: i can't find the pineapple shirt image, did you forgot to upload it?

File: src/features/island/buildings/components/building/smoothieShack/SmoothieShackModal.tsx [hunk 2]
```diff
@@ -49,9 +49,9 @@ export const SmoothieShackModal: React.FC<Props> = ({
       <Panel
         bumpkinParts={{
           body: "Light Brown Farmer Potion",
-          hair: "Explorer Hair",
-          pants: "Lumberjack Overalls",
-          shirt: "Blue Farmer Shirt",
+          hair: "Brown Long Hair",
+          pants: "Farmer Pants",
+          shirt: "Pineapple Shirt",
           tool: "Golden Spatula",
```
Comment (c-py) [inline src/features/island/buildings/components/building/smoothieShack/SmoothieShackModal.tsx:55]:
```suggestion
          tool: "Parsnip",
```

I reckon give old mate a parsnip 😁 

parsnip

**Review by c-py — CHANGES_REQUESTED** [review]

File: src/features/game/types/bumpkin.ts (hunk 1 above)
Comment (mGranTY) [reply src/features/game/types/bumpkin.ts:193]: oh its from the api side.

**Review by c-py — APPROVED** [review]
LGTM!

**Pull request merged**